src/CreatorTokenFactory.sol

```diff
@@ -60,114 +58,10 @@ contract CreatorTokenFactory {
     DeploymentConfig config
   );
 
-  /// @notice Thrown when a deployment fails because it is not verified by the ShowtimeVerifier.
```

Comment:
Similarly I would leave these tests in place and create a new test file for the new factory.